platform/android/src/text/local_glyph_rasterizer_impl.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
```

Comment:
Can we drop the `_impl` in the file name?

Reply:
Yeah, the `_impl` is weird, but what's a better name to use for the Android-specific header? `local_glyph_rasterizer.hpp` is already taken by the shared/portable interface.

I suppose one option would be to just not have a shared hpp file and make each platform responsible for implementing the same `LocalGlyphRasterizer` interface, but it seems nice to keep as much of the interface as possible shared.

Reply:
I'll try `_jni` as a suffix -- I don't see anyone else using that pattern but maybe it's reasonable?

Reply:
We actually use identical header names, in different namespaces, for now. That would be the most consistent. I agree we should probably change that at some point to make navigating the headers easier.